commands/status.go

```diff
@@ -140,18 +140,18 @@ func compare(remoteResourceList *openshift.ResourceList, localResourceList *open
 
 	for _, change := range changeset.Delete {
 		cli.PrintRedf("- %s to delete\n", change.ItemName())
-		fmt.Printf(change.Diff())
+		fmt.Println(change.Diff())
```

Comment:
I think this would add additional newlines, right? I think the equivalent output to `fmt.Printf` would be `fmt.Print`, right?